cypress/integration/stories/opa.spec.ts

```diff
@@ -184,6 +192,9 @@ describe('OPA Story', () => {
   });
 
   it('should add gatekeeper config', () => {
+    if (Mocks.enabled()) {
+      Mocks.gatekeeperConfig = Mocks.defaultGatekeeperConfig;
```

Comment:
Can you try to copy it to the next check too? Perhaps it's overridden somewhere. I'd check that first.

Reply:
yes, already tried, because I thought the same, but nothing changed 😞